packages/vitetris.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Vitetris < Package
+  description 'Classic multiplayer tetris for the terminal'
+  homepage 'https://www.victornils.net/tetris/'
+  version '0.59.1'
+  license 'BSD-2-Clause'
+  compatibility 'all'
+  source_url 'https://github.com/vicgeralds/vitetris.git'
+  git_hashtag "master"
```

Comment:
```suggestion
  git_hashtag "v#{version}"
```

Reply:
Yes, we definitely want to use tagged releases since the master branch can change at any time.